components/EmotionCard.js

```diff
@@ -52,15 +64,39 @@ const StyledLink = styled(Link)`
   }
 `;
 
-const StyledEmotionCard = styled.section`
-  display: flex;
-  flex-direction: column;
-  align-items: left;
-  font-size: 1rem;
+const StyledOuterBox = styled.div`
+  background-color: var(--color-background);
+  border: 1px solid var(--color-border);
+  border-radius: 0.5rem;
+  box-shadow: 0px 1px 4px var(--color-shadow);
```

Comment:
If the value is 0, you can omit the unit.

```suggestion
  box-shadow: 0 1px 4px var(--color-shadow);
```